denops_std/helper/load_test.ts

```diff
@@ -17,10 +17,54 @@ test({
     await load(
       denops,
       new URL(
-        "https://raw.githubusercontent.com/vim-denops/deno-denops-std/main/denops_std/helper/load_test.vim",
+        "https://raw.githubusercontent.com/vim-denops/deno-denops-std/4df988237b3ea04aaa665974b44c8e7df8e237cb/denops_std/helper/load_test.vim",
```

Comment:
Please define a constant variable for this URL.

Reply:
Fixed it with c1fc317.